### PR TITLE
Style updates to Alert and colors 

### PR DESCRIPTION
**Pull request by michellewong793**
Alerts! 
And some updates to theme file for colors in the Figma file. 
Info: 
image

Warning: 
image

### CODE REVIEW

File: src/components/Notice.tsx [hunk 1]
```diff
@@ -25,11 +24,21 @@ export default function Notice(props: Props) {
       css={css(typeToCss(props.type), props.showing ? showingCss : hidingCss)}
       aria-hidden={!props.showing}
     >
+      {returnIcon(props.type)}
       <p>{props.message}</p>
     </div>
   )
 }
 
+function returnIcon(type: Notices) {
```
Comment (aaronmgdr) [inline src/components/Notice.tsx:33]: technically since this returns a react element it could be a component.

**Comment by aaronmgdr** [comment]
im thinking maybe use svg for icons since they are vectors and it will result in them being super crisp at all resolutions?

File: src/components/Notice.tsx (hunk 1 above)
Comment (michellewong793) [reply src/components/Notice.tsx:33]: yeah, i was thinking since it will only really be used for alerts tho to just keep it as a function

**Comment by michellewong793** [comment]
replaced footer icons with svgs too. looks like png just doesnt do well for icons

**Pull request merged**